README.md

```diff
@@ -18,7 +18,7 @@ Supported operating systems (and architectures):
   - RHEL 7.x (x86)
   - RHCOS (x86, IBM Z, and IBM Power Systems)
 
-Full documentation can be found on the [IBM Knowledge Center](https://www.ibm.com/support/knowledgecenter/SSRQ8T).
+Full documentation can be found on the [IBM Knowledge Center](https://www.ibm.com/docs/en/stg-block-csi-driver).
```

Comment:
For full product information, see [IBM block storage CSI driver documentation](https://www.ibm.com/docs/en/stg-block-csi-driver).

Reply:
done